inference-engine/src/transformations/src/transformations/op_conversions/batch_norm_decomposition.cpp

```diff
@@ -58,12 +58,13 @@ ngraph::pass::BatchNormDecomposition::BatchNormDecomposition() {
         auto gamma_div_scale = std::make_shared<opset5::Divide>(m_gamma, scale);
 
         int64_t dims_to_add = m_input.get_partial_shape().rank().get_length() - 2;
-
-        // TODO: instead of getting full shape we can concatenate sequence of ones with ShapeOf
-        Shape input_aligned_shape = m_gamma.get_shape();
-        for (int64_t i = 0; i < dims_to_add; ++i)
-            input_aligned_shape.push_back(1);
-        auto new_shape = opset5::Constant::create(element::i64, Shape{input_aligned_shape.size()}, input_aligned_shape);
+        const auto one = op::Constant::create(element::i64, Shape{1}, {1});
```

Comment:
Should we add new tests to cover the case?

Reply:
BatchNorm single layer tests cover the case